### PR TITLE
Fixes IOU - Unable to paste the amount copied from Notes on MacOS Safari

### PR DESCRIPTION
**Pull request by songlang1994**
### Details

Use `TextInput` directly imported from React Native in `TextInputAutoWidthWithoutKeyboard`. Because `TextInputFocusable` will transform styled text into markdown format when pasting. That's not what we want. See also: https://github.com/Expensify/App/issues/6931#issuecomment-1008097420

### Fixed Issues

$ https://github.com/Expensify/App/issues/6931

### Tests

1. Copy a piece of bold text of amount number format. Some examples is in the following step 2:
2. **12.50** or **1,234.8** or **0.5**
3. Verify they can be pasted into the amount input.

### QA Steps

Same as above.

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

#### Web

web

#### Mobile Web

mobile-web

#### Desktop

desktop

#### iOS

ios

#### Android

ios

### CODE REVIEW

File: src/components/TextInputAutoWidthWithoutKeyboard/BaseTextInputAutoWidthWithoutKeyboard.js [hunk 1]
```diff
@@ -3,12 +3,13 @@ import {
     View,
     AppState,
     Keyboard,
+    // eslint-disable-next-line no-restricted-imports
+    TextInput,
```
Comment (parasharrajat) [inline src/components/TextInputAutoWidthWithoutKeyboard/BaseTextInputAutoWidthWithoutKeyboard.js:7]:
```suggestion
    TextInput as RNTextInput
```

File: src/components/TextInputAutoWidthWithoutKeyboard/BaseTextInputAutoWidthWithoutKeyboard.js [hunk 2]
```diff
@@ -47,7 +48,7 @@ class BaseTextInputAutoWidthWithoutKeyboard extends React.Component {
         return (
             <>
                 <View>
-                    <TextInputFocusable
+                    <TextInput
```
Comment (parasharrajat) [inline src/components/TextInputAutoWidthWithoutKeyboard/BaseTextInputAutoWidthWithoutKeyboard.js:51]:
```suggestion
                    <RNTextInput
```

**Review by parasharrajat — CHANGES_REQUESTED** [review]

**Comment by songlang1994** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by parasharrajat** [comment]
Could you please add screenshots of all the platforms?

**Review by parasharrajat — CHANGES_REQUESTED** [review]
Please fix the styling so that it look the same as before.
|Current|Previous|
|----|----|
|image|image|

File: src/components/TextInputAutoWidthWithoutKeyboard/BaseTextInputAutoWidthWithoutKeyboard.js [hunk 3]
```diff
@@ -3,12 +3,13 @@ import {
     View,
     AppState,
     Keyboard,
+    // eslint-disable-next-line no-restricted-imports
```
Comment (parasharrajat) [inline src/components/TextInputAutoWidthWithoutKeyboard/BaseTextInputAutoWidthWithoutKeyboard.js:6]:
```suggestion

    // eslint-disable-next-line no-restricted-imports
```

**Review by parasharrajat — CHANGES_REQUESTED** [review]

**Review by parasharrajat — APPROVED** [review]
LGTM. 

cc: @Julesssss 

:ribbon: :eyes: :ribbon:  C+ reviewed

**Review by Julesssss — APPROVED** [review]
Tests well.

**Pull request merged**